Teams apps in Outlook and Office preview content (#4554)
MicrosoftDocs/msteams-docs: msteams-platform/m365-apps/overview.md

You can preview your Teams apps running in Microsoft Office and Outlook by updating your code to use the new [Microsoft Teams JavaScript client Preview SDK](using-teams-client-sdk-preview.md) and Microsoft Teams [Developer preview manifest](../resources/schema/manifest-schema-dev-preview.md). With this preview, you can:

- Extend existing Teams [personal tabs](/microsoftteams/platform/tabs/how-to/create-personal-tab) to Outlook for desktop and on the web, and also Microsoft Office Home (office.com).
- Extend existing Teams [search-based messaging extensions](/microsoftteams/platform/messaging-extensions/how-to/search-commands/define-search-command) to Outlook for desktop and on the web.

For feedback and issues, please continue using the relevant [Microsoft Teams developer community channels](/microsoftteams/platform/feedback). We look forward to hearing from you!

## Teams personal tabs in Office and Outlook

With this preview, you can extend a Teams personal tab application to run in both Outlook on Windows desktop and the web, and also Microsoft Office Home (office.com).

> Once sideloaded to Teams, your personal tab will appear as one of your installed apps in Outlook and Office.

Reviewer: ```suggestion
After sideloading to Teams, your personal tab appears as one of your installed apps in Outlook and Office.
```

:::image type="content" source="images/outlook-office-teams-personal-tab.png" alt-text="Personal tab running in outlook.com":::

## Teams messaging extensions in Outlook

Also with this preview, you can extend your search-based Teams messaging extensions to Outlook on the web and Windows desktop, enabling customers to search and share results through the compose message area of Outlook, in addition to Microsoft Teams clients.

> Once sideloaded to Teams, your messaging extension will appear as one of your installed apps within the Outlook compose message area.

Reviewer: ```suggestion
After sideloading to Teams, your messaging extension appears as one of your installed apps within the Outlook compose message area.
```

> ## Next steps

Reviewer: ```suggestion
Next step
```

Author: I think H2 is correct here--it follows the MS Docs recommended structure for overview article types: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-overview?branch=master#next-steps

Reviewer: Hi @erikadoyle, sorry my bad the comment was to change Next Steps to next step

```suggestion
## Next step
```

Author: No worries! I think it is supposed to be Next steps (with an s), per: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-overview?branch=master#next-steps
